data/opponents.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "id": 0,
+    "name": "misty",
```

Comment:
Why do you write the names in lowercase here?

Reply:
there is actually no reason for that, I have a `text-transform: uppercase` in the CSS classes that use that name since; so I didn't really pay attention to that. Nevertheless adapted!